tests/deployment_utils.py

```diff
@@ -78,15 +95,28 @@ def assert_unsuccessful_execution(self, cmd, environment=None):
 
     def install_python_module(self, name):
```

Comment:
Does it make sense to always attempt to install modules, even when new container is not created?